test/fuzzTesting/createRandomStateTest.cpp

```diff
@@ -47,55 +48,7 @@ void doStateTests(json_spirit::mValue& _v);
 int main(int argc, char *argv[])
 {
 	g_logVerbosity = 0;
-
```

Comment:
instead of completely removing the old code (which works very nicely), please implement it as algorithm1, algorithm2, ... .